packages/components/src/components/common/inputbadge/inputBadge.tsx

```diff
@@ -23,24 +22,22 @@ const Badge = ({ icon, text }: { icon: IvyIcons; text: string }) => (
   </Flex>
 );
 
-export const InputBadge = forwardRef<ElementRef<'output'>, InputBadgeProps>(
-  ({ value, badgeProps, className, style, ...props }, forwardRef) => {
-    const { inputProps } = useField();
-    const items = useMemo(() => findBadges(value, badgeProps), [value, badgeProps]);
-    return (
-      <output className={`${inputBadgeOutput} ${className}`} style={style} {...inputProps} {...props} ref={forwardRef}>
```

Comment:
if you do it like this and className is undefined, undefined gets added as string to the dom
use the existing cn helper function for stuff like that

Reply:
:+1: good to know